image/tarexport/save.go

```diff
@@ -269,6 +331,37 @@ func (s *saveSession) save(outStream io.Writer) error {
 		return err
 	}
 
+	// Write the index.json file
+	// This export may contain multiple totally different images
+	// As such we'll only create the index.json if there is only 1 image.
+	if len(manifest) == 1 {
```

Comment:
Shouldn't we do this even with multiple images included?  `ctr` does: :eyes:

```json
{
  "schemaVersion": 2,
  "manifests": [
    {
      "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
      "digest": "sha256:aa0cc8055b82dc2509bed2e19b275c8f463506616377219d9642221ab53cf9fe",
      "size": 2561,
      "annotations": {
        "io.containerd.image.name": "docker.io/library/hello-world:latest",
        "org.opencontainers.image.ref.name": "latest"
      }
    },
    {
      "mediaType": "application/vnd.oci.image.manifest.v1+json",
      "digest": "sha256:24bb4d8133f682cc140769253142a636b834eb9cde17fec36f5a53b0adec3d5e",
      "size": 438,
      "annotations": {
        "io.containerd.image.name": "docker.io/tianon/scratch:gzipzerobytes",
        "org.opencontainers.image.ref.name": "gzipzerobytes"
      }
    }
  ]
}
```

Reply:
Yeah, I would really expect `index.json` to always exist and contain all the roots/tags.

Reply:
Updated this so the behavior is the same as the containerd exporter.